custom_components/powercalc/data/signify/1743530P7/model.json

```diff
@@ -0,0 +1,16 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly, Plus1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "master"
+    },
+    "aliases": [
+        "17435/30/P7"
```

Comment:
How is the light recognised in your HA installation? With `1743530P7` or `17435/30/P7`? I never seen the latter in any of the hue lights, and I also foresee some issues because there is some magic with slashes in the powercalc matching logic. Needs more thorough testing to see if we can handle this case without any problems down the road.
So when in your HA installation `1743530P7` is used I suggest to just remove these aliases as it would never be used.

Reply:
Het lijkt erop dat hij met slashes gebruikt: (bij al mijn lampen gebruikt hij herstellers model nummer ipv zigbee model nummer.
![image](https://user-images.githubusercontent.com/11457346/218303452-974b118e-36b8-472f-98ab-ad30dc8bbb32.png)

Reply:
I removed the alias for now.

Reply:
hmm this the light is reported as `17435/30/P7` in HA, so that means we need to have the alias in order for auto discovery to work correctly. I will add some tests in the codebase to see if this works correctly.
So you may re-add the alias.

Reply:
Already had a test in place for this one: https://github.com/bramstroker/homeassistant-powercalc/blob/master/custom_components/powercalc/data/ikea/LED1649C5/model.json#L9.
This case is similar so I don't foresee any issues.

Reply:
Ok, I've added it back again.